cli/args/flags.rs

```diff
@@ -717,6 +718,12 @@ fn clap_root(version: &str) -> Command {
 fn bench_subcommand<'a>() -> Command<'a> {
   runtime_args(Command::new("bench"), true, false)
     .trailing_var_arg(true)
+    .arg(
+      Arg::new("json")
+        .long("json")
+        .help("Output benchmark result in JSON format")
```

Comment:
Add `UNSTABLE:` to the help string for now.

Reply:
Done